PublishWorkflow/PublishWorkflowChecker.php

```diff
@@ -166,4 +163,32 @@ public function isGranted($attributes, $object = null)
 
         return $this->accessDecisionManager->decide($token, $attributes, $object);
     }
+
+    private function getTokenStorage()
+    {
+        if (null === $this->tokenStorage) {
+            if ($this->container->has('security.token_storage')) {
+                $this->tokenStorage = $this->container->get('security.token_storage');
+            } else {
```

Comment:
we could call this method getTokenFromStorage and set $this->tokenStorage to `false` if both `has` return false, and detect that situation to return null.